client/db.json

```diff
@@ -8,53 +8,25 @@
       "id": 0
     },
     {
-      "lng": "-3.869932960937492",
-      "lat": "55.4045715267554",
-      "title": "uigp",
-      "description": "uy9",
+      "text": "fdbdbd",
```

Comment:
would we be able to retain the naming convention to "title"?

Reply:
On it